src/lib.rs

```diff
@@ -163,7 +165,7 @@ impl Dependencies {
     /// A set of dependencies to run against.
     pub fn new(execution_config: ExecutionConfiguration) -> Result<Self> {
         let fs_service = Arc::new(FsServiceImpl::new());
-        let discovery_service = Arc::new(ResmokeProxy::new(execution_config.resmoke_command));
+        let discovery_service = Arc::new(ResmokeProxy::new(execution_config.resmoke_command, execution_config.is_patch));
```

Comment:
could you double check the linting and testing [here](https://github.com/mongodb/mongo-task-generator)?

Reply:
Looks like I missed making this change - I've done it now!

Btw the [README](https://github.com/mongodb/mongo-task-generator?tab=readme-ov-file#lintingformatting) doesn't really explain how to run lint / format locally. The suggested commands don't work:
```
ubuntu@ip-10-122-6-249:~/mongo-task-generator$ cargo fmt
error: no such command: `fmt`

	Did you mean `fix`?

	View all installed commands with `cargo --list`
	Find a package to install `fmt` with `cargo search cargo-fmt`
```

Reply:
Hm, that's weird. My `cargo` has this command, but I don't remember setting anything up for that to work.
Anyway PR checks are green so we should be good.